game/lobby.go

```diff
@@ -621,8 +622,13 @@ func triggerPlayersUpdate(lobby *Lobby) {
 	TriggerComplexUpdateEvent("update-players", lobby.Players, lobby)
 }
 
-func triggerCorrectGuessEvent(lobby *Lobby) {
-	TriggerSimpleUpdateEvent("correct-guess", lobby)
+// Broadcast `word-guessed-message` event to all `lobby` players about `player` success
+func triggerCorrectGuessEvent(lobby *Lobby, player *Player) {
+	SendDataToConnectedPlayers(
+		&Player{},
+		lobby,
+		JSEvent{Type: "word-guessed-message", Data: fmt.Sprintf("%s guessed the word!", player.Name)},
```

Comment:
We might actually just make an event that contains the player name and ID or so. That way the client can decide on how to render. Scribble.rs now has a usable public API. I want to keep it as usable as possible.